.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
   global:
   - TRAVIS_NODE_VERSION="8.9"
 install:
+- curl https://raw.githubusercontent.com/scala-native/scala-native/master/scripts/travis_setup.sh | bash -
```

Comment:
In the Scalameta build we run native tests on a separate job because native installation takes a while https://github.com/scalameta/scalameta/blob/ae6a9a73032e8c2225e7b08751e0a5c29fcbd74b/.travis.yml#L29-L34 

From the logs this step took 150s
<img width="1035" alt="screen shot 2018-06-29 at 09 43 08" src="https://user-images.githubusercontent.com/1408093/42079774-e525bacc-7b80-11e8-93e0-20800e9d2b1e.png">